mysqldefaults.conf

```diff
@@ -0,0 +1,3 @@
+[client]
+user=root
+password=root
```

Comment:
Would this be `db:db` now?

Reply:
This file should not even be added to the repository here. We could add it to `.gitignore`, but not just create one.